package.json

```diff
@@ -38,10 +38,11 @@
   "scripts": {
     "debug": "node debug app.js",
     "start": "node app.js",
-    "test": "NODE_ENV=test mocha test/bootstrap.test.js test/integration/**/*.test.js"
+    "test": "NODE_ENV=test ./node_modules/.bin/mocha test/bootstrap.test.js test/integration/**/*.test.js"
```

Comment:
Doesn't really make a difference here but more as a fyi - I think as long as `mocha` (or any other package) is in `node_modules`, you can use `mocha` globally in `npm scripts` without specifying `./node_modules/.bin/